decidim-admin/app/commands/decidim/admin/update_organization_appearance.rb

```diff
@@ -50,6 +58,7 @@ def attributes
           .merge(highlighted_content_banner_attributes)
           .merge(omnipresent_banner_attributes)
           .merge(colors_attributes)
+          .delete_if { |_k, val| val.is_a?(Decidim::ApplicationUploader) }
```

Comment:
Why this?

Reply:
Because in the class initializer we assign the object logo to the form. We assign it, as if we do not perform that step, the images in the form will be lost when updating other fields. 
In this particular case we delete the file to prevent multiple processing of the same file. This is a workaround that i have found in decidim codebase, added via: #7026

Reply:
Got it, thanks!